cmd/nerdctl/volume_ls.go

```diff
@@ -154,3 +164,54 @@ func getVolumes(cmd *cobra.Command) (map[string]native.Volume, error) {
 	}
 	return volStore.List(volumeSize)
 }
+
+func getFilters(filters []string) ([]map[string]string, []string, bool) {
+	isFilter := len(filters) > 0
+	labelFilters := make([]map[string]string, 0)
```

Comment:
Maybe making this as a function should be a good idea

So you can simply use it in here https://github.com/containerd/nerdctl/pull/1331/files#diff-5cd0eb713e85375bd7c5ec6da2b72eb42ebaafe5c03f582ffbbf77ee60a02b26R200-R208 like code below


```go
	for _, filterLabelFunc := range filterLabelFuncs {
                filterLabelFunc(v.Labels)
	}
```